tools/terraform/redis.tf

```diff
@@ -2,7 +2,7 @@ variable "maintenance_window" {
   default = ""
 }
 variable "engine_version" {
-  default = "6.x"
+  default = "5.0.6"
```

Comment:
we don't want to use redis 6?

Reply:
We can, this is just reflecting what is in the loadtesting environment currently.